### PR TITLE
feat: added cart management

### PR DESCRIPTION
**Pull request by AnjaliDeshwani**
**This PR contains the below features-**

- Add to Cart from Product Listing page
- Add to Cart from Wishlist
- Increase and Decrease the quantity of products
- Remove from Cart, Move to Wishlist from Cart, Move to Cart from Wishlist
- Price Details according to cart items

### CODE REVIEW

File: src/pages/Cart/Cart.css [hunk 1]
```diff
@@ -61,11 +65,44 @@
   margin-bottom: 4px;
   font-size: 0.875rem;
 }
+.cart-item .coupon-details {
+  color: #2057a8;
```
Comment (muskaanshah) [inline src/pages/Cart/Cart.css]: Suggestion: use variables

**Review by mohammadkhizerkhan — APPROVED** [review]
everything seems good to me

**Review by muskaanshah — APPROVED** [review]
Suggestions:
1. To add a tab for products in the navbar, user cant navigate to products without going to home page
2. Text on the home banner isn't seen, add a black overlay to the image so that text is visible
3. Buttons and category headings are not clickable, on clicking it should navigate to the products page

Everything else looks good, functionality working properly

File: src/pages/Cart/Cart.css (hunk 1 above)
Comment (AnjaliDeshwani) [reply src/pages/Cart/Cart.css]: I have updated it now. Thanks for pointing it out.

**Comment by AnjaliDeshwani** [comment]
> Suggestions:
> 
> 1. To add a tab for products in the navbar, user cant navigate to products without going to home page
> 2. Text on the home banner isn't seen, add a black overlay to the image so that text is visible
> 3. Buttons and category headings are not clickable, on clicking it should navigate to the products page
> 
> Everything else looks good, functionality working properly

**Comment by AnjaliDeshwani** [comment]
> Suggestions:
> 
> 1. To add a tab for products in the navbar, user cant navigate to products without going to home page
> 2. Text on the home banner isn't seen, add a black overlay to the image so that text is visible
> 3. Buttons and category headings are not clickable, on clicking it should navigate to the products page
> 
> Everything else looks good, functionality working properly

Thanks @muskaanshah for thorough review. I still need to work on home page will cover all these things in the next PR.

**Pull request merged**